modules/installation-special-config-mirrored-disk.adoc

```diff
@@ -87,3 +87,67 @@ $ podman run --pull=always --rm --volume $HOME/clusterconfig:/pwd --workdir /pwd
 This command produces a combined Ignition config in `$HOME/clusterconfig/worker-raid.ign`.
 +
 . Continue with the remainder of the {product-title} deployment, using the combined Ignition config to provision nodes.
+
+== Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume.
+
+. Create a machine config that configures a data volume by using software RAID.
+
+* To configure a data volume with RAID 1 on secondary disks, save the following machine config sample to a file in the `<installation_directory>/openshift` directory. For example, name the file `raidl-alt-storage.yaml`:
++
+.RAID 1 on secondary disks
+[source, yaml]
+----
+variant: openshift
+version: 4.7.0
+metadata:
+  name: raidl-alt-storage
+  labels:
+    machineconfiguration.openshift.io/role: worker
+storage:
+  disks:
+    - device: /dev/sdc
+      wipeTable: true
+      partitions:
+        - label: data-1
+    - device: /dev/sdd
+      wipeTable: true
+      partitions:
+        - label: data-2
+  raid:
+    - name: md-var-lib-containers
+      level: raid1
+      devices:
+        - /dev/disk/by-partlabel/data-1
+        - /dev/disk/by-partlabel/data-2
+  filesystems:
+    - device: /dev/md/md-var-lib-containers
+      path: /var/lib/containers
+      format: xfs
+      wipeFilesystem: true
+systemd:
+  units:
+    - contents: |
+        [Unit]
+        Before=local-fs.target
+        Requires=systemd-fsck@dev-md-md\x2dvar\x2dlib\x2dcontainers.service
+        After=systemd-fsck@dev-md-md\x2dvar\x2dlib\x2dcontainers.service
+
+        [Mount]
+        Where=/var/lib/containers
+        What=/dev/md/md-var-lib-containers
+        Type=xfs
+
+        [Install]
+        RequiredBy=local-fs.target
+      enabled: true
+      name: var-lib-containers.mount
+----
+. Apply the config to your cluster by running the following command:
++
+[source, terminal]
+----
+$ oc create -f raidl-alt-storage.yaml
+----
+. Save the machine config in case you need to update it in the future. 
```

Comment:
This line is meant for Butane configs, which aren't stored in the cluster.  I think we can drop it when writing a MachineConfig directly.